tests/unit/v2/API/metrics/testIngestMetric.js

```diff
@@ -0,0 +1,38 @@
+const assert = require('assert');
+const sinon = require('sinon');
+const ingestMetric = require('../../../../../libV2/server/API/metrics/ingestMetric');
+const { client: cacheClient } = require('../../../../../libV2/cache');
+
+const { generateFakeEvents, templateContext } = require('../../../../utils/v2Data');
+
+const events = generateFakeEvents(1, 50, 50);
+
+describe('Test ingestMetric', () => {
+    let ctx;
+    beforeEach(() => {
+        ctx = templateContext();
+    });
+
+    afterEach(() => sinon.restore());
```

Comment:
I think this also works
```suggestion
    afterEach(sinon.restore);
```

Reply:
sinon looses some sort of internal state without the `() =>` and throws an error

Reply:
hmm weird, will have to remember this for later